.github/workflows/integration-test.yml

```diff
@@ -49,14 +49,73 @@ jobs:
       - name: Register Operator
         run: |
           cd eigenlayer-cli
-          echo "" | ./bin/eigenlayer operator register operator-ci.yaml
+          echo "" | ./bin/eigenlayer operator register tests/keystore/operator-ci.yaml
 
       - name: Check operator status
         run: |
           cd eigenlayer-cli
-          ./bin/eigenlayer operator status operator-ci.yaml
+          ./bin/eigenlayer operator status tests/keystore/operator-ci.yaml
 
       - name: Update operator details
         run: |
           cd eigenlayer-cli
-          echo "" | ./bin/eigenlayer operator update operator-ci.yaml
\ No newline at end of file
+          echo "" | ./bin/eigenlayer operator update tests/keystore/operator-ci.yaml
+  Web3Signer:
+    name: Integration Test - Web3 Signer
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.21'
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+      - name: Checkout incredible squaring
+        uses: actions/checkout@v4
+        with:
+          repository: layr-labs/incredible-squaring-avs
+          token: ${{ github.token }}
+          ref: master
+      - name: Run anvil chain
+        run: |
+          nohup make start-anvil-chain-with-el-and-avs-deployed > nohup.out 2>&1 &
```

Comment:
isn't there a way to cache and/or share those things between the 2 jobs? not sure if this is the kind of costs that are worth optimizing for

Reply:
hmm I want to keep 2 test isolated - also they run in parallel - time wise it's fine. but I see what you are saying. I don't think we want to optimize it right now - it's quite fast anyway. If I keep adding more tests like this in future then I can explore if we can share these between jobs.